master/getting-started/kubernetes/installation/hosted/kubeadm/index.md

```diff
@@ -11,6 +11,13 @@ own etcd.
 
 You can easily create a cluster compatible with this manifest by following [the official kubeadm guide](http://kubernetes.io/docs/getting-started-guides/kubeadm/).
 
+> Notes:
+>   - The Kubeadm flag `--cluster-cidr` (pre 1.6) or
+>     `--pod-network-cidr` (1.6+) is not necessary when using Calico.
```

Comment:
"is not necessary when using Calico" -> "is not utilized by Calico nor Kubernetes if using Calico for routing"